output/output.go

```diff
@@ -138,29 +140,43 @@ func PrintCmdOutput(cmd *cobra.Command, v any) {
 // If cmd is not provided or it has no `output` flag, human is assumed
 // If human format is requested/assumed but no table is provided, displays YAML
 // If the object cannot be converted to the desired format, shows the object in Go's %+v format
-func PrintCmdOutputCustom(cmd *cobra.Command, v any, table *Table) {
-	// extract format, assume default if no command or no -o flag
-	format := ""
-	if cmd != nil {
-		format, _ = cmd.Flags().GetString("output") // if err, leaves format blank
+func PrintCmdOutputCustom(cmd *cobra.Command, v any, pr *PrintRequest) {
```

Comment:
I would prefer not to break backward compatibility and complicate the invocation for callers. Can you either (a) move PrintRequest inside the Table (as an option, nil by default) or (b) create a PrintCmdOutputAdvanced or PrintCmdOutputRequest or similar?

This will remove the need to change other commands that use PrintCmdOutputCustom

Reply:
Done https://github.com/cisco-open/fsoc/blob/e3338c589daffddf0f80facb966729c53d87cad0/output/output.go#L138-L147